packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/Choices/Choices.js

```diff
@@ -1,73 +1,123 @@
 import {
-  Box,
-  Grid
+  Box
 } from 'grommet'
 import PropTypes from 'prop-types'
-import React from 'react'
-import sortIntoColumns from 'sort-into-columns'
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
 import howManyColumns from './helpers/howManyColumns'
 import whatSizeThumbnail from './helpers/whatSizeThumbnail'
 import ChoiceButton from './components/ChoiceButton'
 
+const StyledGrid = styled(Box)`
```

Comment:
This is overwriting the `Box` flex display settings, so it's not really needed to use `Box` here. Just use a div instead?
```suggestion
const StyledGrid = styled.div`
```